src/sentry/integrations/jira/integration.py

```diff
@@ -344,7 +344,7 @@ def get_client(self):
 
         return JiraApiClient(
             self.model.metadata["base_url"],
-            JiraCloud(self.model.metadata["shared_secret"]),
```

Comment:
Instead of instantiating `JiraCloud`, you pass the shared secret directly to `JiraApiClient` - this was also a part of the `jira_style` removal.